services/frontend/src/components/admin/userTable/UserTableView.less

```diff
@@ -12,3 +12,8 @@
   margin-left: 8px;
   padding-right: 10px;
 }
+
+.icon {
+  text-align: center;
+  font-size: 150%;
+}
```

Comment:
be careful about using such a generic className since it will effect other parts of the project. 
i would either nest it inside another className or make a more specific name to avoid a conflict in styling

Reply:
Good point fixed